generator.yaml

```diff
@@ -865,3 +866,5 @@ ignore:
       - Workteam
     operations:
       - UpdateTrainingJob
+    shape_names:
+      - RSessionAppSettings
```

Comment:
I always find it useful to add a comment above these types of additions, just to let readers (or future me...) know why I did certain things. For example, here, I'd put something to the effect of:

```yaml
      # This is an empty struct that causes generation errors
```